source/codegen/common_helpers.py

```diff
@@ -163,20 +163,20 @@ def filter_proto_rpc_functions(functions):
 
 
 def get_attribute_enums_by_type(attributes):
```

Comment:
(Sorry) Just an indentation change. This method was out-of-sync with the rest of the file and was causing problems.